sample-app/src/components/DropdownSection.tsx

```diff
@@ -29,12 +29,13 @@ export default function DropdownSection({
   optionIdPrefix,
   onFocusChange = () => {},
   onLeaveSectionFocus = () => {},
-  onClickOption = () => {},
+  onSelect = () => {},
   label = '',
   cssClasses
 }: DropdownSectionProps): JSX.Element | null {
 
   const [focusedOptionIndex, setFocusedOptionIndex] = useState<number>(0);
+  const wasActive = useRef<boolean>(focusStatus === 'active');
```

Comment:
this is a little out of scope but is focusStatus ever anything but 'active' or 'inactive'? could it just be an `isFocused` boolean?

Reply:
good catch. there used to be a third status, but now that there are only two, we can make it a boolean

Reply:
updated!